newIDE/app/src/UI/ClosableTabs.js

```diff
@@ -192,12 +189,15 @@ export function ClosableTab({
               >
                 <span
                   style={{
-                    ...styles.tabLabel,
+                    ...styles.tabLabelAndIcon,
+                    height: muiTheme.closableTabs.height,
                     color: textColor,
                     fontFamily: muiTheme.closableTabs.fontFamily,
                   }}
                 >
-                  {label}
+                  {icon}
```

Comment:
for accessibility purpose, should a label be used as an alt/title for the icon/tab?

Reply:
Great remark, I've added a title to the Home icon, which will give a tooltip and an equivalent text for screen readers ("Provides a human-readable title for the element that contains it. The title element may be rendered by a graphical user agent as a tooltip. It may be rendered as speech by a speech synthesizer.")